lib/ProductOpener/Routing.pm

```diff
@@ -545,6 +551,7 @@ sub facets_route($request_ref) {
 	}
 
 	$request_ref->{canon_rel_url} .= $canon_rel_url_suffix;
+	$request_ref->{rate_limiter_bucket} = "facet";
 
 	if (defined $request_ref->{groupby_tagtype}) {
 		set_request_stats_value($request_ref->{stats}, "route", "facets_tags");
```

Comment:
Maybe set the rate_limiter_bucket to facets_tags. Those are very different queries: currently the list of products "facets" goes to mongodb, and the list of tags "facets_tags" goes to off-query

Reply:
Right, I will add a distinct bucket for `facet_tags`.